Src/BiometryService/BiometryService.UWP.cs

```diff
@@ -0,0 +1,38 @@
+﻿#if WINDOWS_UWP
```

Comment:
I would name this BiometryService.NotImplemented.cs or something similar. No need to tie it to a platform